src/EprPrnIntegration.Common/Mappers/NpwdPrnToSavePrnDetailsRequestMapper.cs

```diff
@@ -0,0 +1,62 @@
+﻿using EprPrnIntegration.Common.Constants;
+using EprPrnIntegration.Common.Models;
+
+namespace EprPrnIntegration.Common.Mappers
+{
+
+    public static class NpwdPrnToSavePrnDetailsRequestMapper
+    {
+        public static SavePrnDetailsRequest Map(NpwdPrn npwdPrn)
```

Comment:
use of automapper?

also it can be used as an expression method

Reply:
Note, we have been discouraged from using AutoMapper in .Net 8

Reply:
Yes I am told to use manual mapping